.github/workflows/Formatter.yml

```diff
@@ -0,0 +1,37 @@
+name: Formatter
```

Comment:
wasn't there already a format yml?

Reply:
The workflow in #740 is to check if a PR complies to the style guide, but this workflow is for continuous checking the main branch and creates a PR automatically if needed.

Reply:
We settled on not doing that so that the git blame would more readily point back to the correct PR